### DISCUSSION TITLE
How to assign specified pores and throats

### DISCUSSION THREAD

**Discussion by Devin-one** [PMEAL/OpenPNM#2905] [Q&A]
Hello professor, I would like to set two types of pore size distributions on a two-dimensional cubic network. The left side of the network is assigned pore diameters of the same size and throat diameters of the same size. The right side of the network is also assigned pore diameters of the same size and throat diameters of the same size. How should I operate it? Thank you in advance for your reply.

**Comment by jgostick** [comment]
There are a few examples which might help get you started.
1. [This one](https://openpnm.org/examples/applications/adjusting_pore_size_distributions.html) explains how to assigned different pore size distributions
2. [This one](https://openpnm.org/examples/tutorials/06_labels_and_domains.html) explains how to assign labels to certain pores, which you can then use to assign different models to different regions of the network.
